src/opnsense/service/templates/OPNsense/Proxy/squid.acl.conf

```diff
@@ -99,6 +99,16 @@ http_access deny blockmimetypes_requests {% if helpers.exists('OPNsense.proxy.fo
 
 {% endif %}
 
+# Google Suite Filter
+{% if helpers.exists('OPNsense.proxy.forward.acl.googleapps') and OPNsense.proxy.forward.acl.googleapps|default('') != '' %}
+request_header_add X-GoogApps-Allowed-Domains {{OPNsense.proxy.forward.acl.googleapps}} localnet
```

Comment:
is localnet needed here? most access rules don't restrict to localnet, in which case the same policy would also apply for routed networks.

Reply:
In this case, what do you recommend to use here?

Reply:
@juliocbc If I'm not mistaken you can omit the acl, in which case it will match on _all_ policies (http://www.squid-cache.org/Doc/config/request_header_add/)

Reply:
@AdSchellevis you are right! I will change it! Thanks!